code/modules/mob/living/simple_animal/bot/griefsky.dm

```diff
@@ -207,6 +207,26 @@
 
 //this section is blocking attack
 
+///Disassembling the bot in a civilized manner with a multitool
+/mob/living/simple_animal/bot/secbot/griefsky/disassemble()
+	walk_to(src,0)
```

Comment:
Same here, do we need this?

Reply:
Nope! I removed it and did not notice any problem while interrupting bot movement / actions by disassembling them. It is probably safe to remove from the `explode()` code as well.